pkg/arvo/mar/contact/update.hoon

```diff
@@ -1,15 +1,32 @@
-/+  *contact-json
-|_  upd=contact-update
+/+  *contact-store
+::
+|_  upd=update
 ++  grad  %noun
 ++  grow
   |%
   ++  noun  upd
-  ++  json  (update-to-json upd)
+  ++  json  (update:enjs upd)
+  ++  resource
+    |^
+    ?-  -.upd
+      %initial     [nobody %contacts]
```

Comment:
this will clash with the `src.bowl` checking that we do now. Maybe we could special case the `nobody` entity?